### PR TITLE
Turkish translation

### PR DESCRIPTION
**Pull request by haktanemik**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [X] Your PR is linked to an issue.
- [X] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [X] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [X] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR covers issue #560 .

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

**Review by cpholguera — APPROVED** [review]
This is awesome @haktanemik! What an excellent surprise. Thanks to your contribution we cover now 14 languages 🎉 

Thank you very much for the hard work, once we merge it we will announce it on Twitter, LinkedIn and Slack. Would you like us to mention you directly? If yes, please send us your handles for the different platforms.

**Comment by haktanemik** [comment]
Hi @cpholguera,

Thanks a lot for fast and kind your response. We would be very pleased if you mention about us. We made this translation with my friend Anıl.

Twitter usernames are @haktanemik and @0x4n1l
Linkedin usernames are @haktanemik and @anilbas1

Thank you very much.

**Comment by cpholguera** [comment]
That's awesome, we will definitely mention you both!

The process goes like this: now we require a second approval from an independent target language speaker. We're currently searching for one. After that we'll merge and announce :)

Meanwhile I just need you to do one little thing. Could you please `git pull origin master` and then `git push`. I've fixed the linting and URL checking errors and with this your PR will show all green. Thanks in advance!

And thanks a lot to Anıl as well!

**Comment by Glassware123** [comment]
Hi @cpholguera,

I've fixed markdown issues. Thanks in advance!

Regards,
Anıl

File: Document-tr/0x01-Foreword.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+MASVS genel olarak, mobil uygulama güvenliği için bir temel sunmak (MASVS-L1), aynı zamanda derinlemesine savunma önlemlerinin (MASVS-L2) ve istemci taraflı tehditlere (MASVS-R) karşı korumaların dahil edilmesini hedefler. MASVS, aşağıdakileri gerçekleştirmeyi amaçlar:
+
+- Güvenli mobil uygulamalar geliştirmek isteyen geliştiriciler ve yazılım mimarları için gereksinimleri sağlamak;
+- Mobil uygulama güvenlik incelemelerinde test edilebilecek bir endüstri standardı sunmak;
+- Yazılım koruma mekanizmalarının mobil güvenlikteki rolünü netleştirmek ve etkinliklerini doğrulamak için gereksinimleri sağlamak;
+- Farklı kullanım durumları için hangi güvenlik seviyesinin tavsiye edildiğine dair spesifik öneriler sunmak.
+
+%100 endüstri mutabakatına ulaşmanın imkansız olduğunun farkındayız. Yine de, MASVS'nin mobil uygulama geliştirme ve testlerinin tüm aşamalarında rehberlik sağlamada yararlı olacağını umuyoruz. Açık kaynak standardı olarak MASVS zamanla gelişecek ve her türlü katkı ve öneriyi memnuniyetle karşılıyoruz.
+
+Bernhard Mueller tarafından
```
Comment (cpholguera) [inline Document-tr/0x01-Foreword.md:26]:
```suggestion
Bernhard Mueller tarafından

```

File: Document-tr/0x02-Frontispiece.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+### "Honourable Benefactor" Sponsoru
+
+[![NowSecure](images/NowSecure_logo.png)](https://www.nowsecure.com/)
+
+### "Good Samaritan Benefactor" Sponsoru
+
+[![Randorisec](images/Randorisec_logo.png)](https://www.randorisec.fr/)
+
+Ardından, sponsorlukları için OWASP Körfez Bölgesi Bölümüne teşekkür ederiz. Son olarak, kitabı [Leanpub'dan](https://leanpub.com/mobile-security-testing-guide) alan ve bize sponsor olan herkese teşekkür ederiz.
```
Comment (cpholguera) [inline Document-tr/0x02-Frontispiece.md:59]:
```suggestion
Ardından, sponsorlukları için OWASP Körfez Bölgesi Bölümüne teşekkür ederiz. Son olarak, kitabı [Leanpub'dan](https://leanpub.com/mobile-security-testing-guide) alan ve bize sponsor olan herkese teşekkür ederiz.

```

File: Document-tr/0x03-Using_the_MASVS.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+###### MASVS L2+R
+
+- Finansal Endüstri: Ele geçirilen cihazlarda kod enjeksiyonu ve enstrümantasyon gibi tekniklerin risk oluşturduğu, para transferleri yapılabilen çevrimiçi bankacılık uygulamalarının uygulaması önerilir. Bu durumda, MASVS-R kontrolleri, kurcalanmayı engellemek için kullanılabilir ve kötü amaçlı yazılım sahipleri için çıtayı yükseltir.
+
+- Tasarım gereği hassas verileri mobil cihazda depolaması gereken ve aynı zamanda çok çeşitli cihazları ve işletim sistemi sürümlerini desteklemesi gereken tüm mobil uygulamalarının uygulaması önerilir. Bu durumda dayanıklılık kontrolleri, hassas verileri çıkarmayı amaçlayan saldırganların çabalarını artırmak için kapsamlı bir savunma önlemi olarak kullanılabilir.
+
+- Uygulama içi satın alımları olan uygulamalar, ücretli içeriği korumak için ideal olarak sunucu tarafı ve MASVS-L2 kontrollerini kullanmalıdır. Ancak, sunucu tarafı korumasını kullanma olanağının olmadığı durumlar olabilir. Bu durumlarda, tersine mühendislik ve/veya kurcalama çabasını artırmak için MASVS-R kontrolleri ek olarak uygulanmalıdır.
```
Comment (cpholguera) [inline Document-tr/0x03-Using_the_MASVS.md:85]:
```suggestion
- Uygulama içi satın alımları olan uygulamalar, ücretli içeriği korumak için ideal olarak sunucu tarafı ve MASVS-L2 kontrollerini kullanmalıdır. Ancak, sunucu tarafı korumasını kullanma olanağının olmadığı durumlar olabilir. Bu durumlarda, tersine mühendislik ve/veya kurcalama çabasını artırmak için MASVS-R kontrolleri ek olarak uygulanmalıdır.

```

File: Document-tr/0x04-Assessment_and_Certification.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+### Otomatik Birim ve Entegrasyon Testleri için Kılavuz Olarak
+
+MASVS, mimari gereksinimler dışında, yüksek düzeyde test edilebilir olacak şekilde tasarlanmıştır. MASVS gereksinimlerine dayalı otomatik birim, entegrasyon ve kabul testi, sürekli geliştirme yaşam döngüsüne entegre edilebilir. Bu yalnızca geliştirici güvenlik farkındalığını artırmakla kalmaz, aynı zamanda ortaya çıkan uygulamaların genel kalitesini iyileştirir ve devreye alım öncesi aşamada güvenlik testi sırasında bulguların miktarını azaltır.
+
+### Güvenli Geliştirme Eğitimi İçin
+
+MASVS, güvenli mobil uygulamaların özelliklerini tanımlamak için de kullanılabilir. Çoğu "güvenli kodlama" kursu, üstünkörü bir şekilde kodlama ipuçlarına değinerek basitçe etik bilgisayar korsanlığı kurslarıdır. Bu, geliştiricilere yardımcı olmaz. Bunun yerine, güvenli geliştirme kursları örneğin en sık karşılaşılan 10 kod güvenlik bulgusuna odaklanmak yerine MASVS'de belgelenen proaktif kontrollere güçlü bir odaklanma ile MASVS'yi kullanabilir.
```
Comment (cpholguera) [inline Document-tr/0x04-Assessment_and_Certification.md:49]:
```suggestion
MASVS, güvenli mobil uygulamaların özelliklerini tanımlamak için de kullanılabilir. Çoğu "güvenli kodlama" kursu, üstünkörü bir şekilde kodlama ipuçlarına değinerek basitçe etik bilgisayar korsanlığı kurslarıdır. Bu, geliştiricilere yardımcı olmaz. Bunun yerine, güvenli geliştirme kursları örneğin en sık karşılaşılan 10 kod güvenlik bulgusuna odaklanmak yerine MASVS'de belgelenen proaktif kontrollere güçlü bir odaklanma ile MASVS'yi kullanabilir.

```

File: Document-tr/0x06-V1-Architecture_design_and_threat_modelling_requireme.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+## Referanslar
+
+Daha fazla bilgi için ayrıca bakınız:
+
+- OWASP Mobile Top 10: M10 (Extraneous Functionality) - <https://owasp.org/www-project-mobile-top-10/2016-risks/m10-extraneous-functionality>
+- OWASP Threat modelling - <https://owasp.org/www-community/Application_Threat_Modeling>
+- OWASP Secure SDLC Cheat Sheet - <https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets_excluded/Secure_SDLC_Cheat_Sheet.md>
+- Microsoft SDL - <https://www.microsoft.com/en-us/sdl/>
+- NIST SP 800-57 - <https://csrc.nist.gov/publications/detail/sp/800-57-part-1/rev-5/final>
+- security.txt - <https://securitytxt.org/>
```
Comment (cpholguera) [inline Document-tr/0x06-V1-Architecture_design_and_threat_modelling_requireme.md:37]:
```suggestion
- security.txt - <https://securitytxt.org/>

```

File: Document-tr/0x07-V2-Data_Storage_and_Privacy_requirements.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+- CWE 359 (Exposure of Private Information ('Privacy Violation')) - <https://cwe.mitre.org/data/definitions/359.html>
+- CWE 522 (Insufficiently Protected Credentials) - <https://cwe.mitre.org/data/definitions/522.html>
+- CWE 524 (Information Exposure Through Caching) - <https://cwe.mitre.org/data/definitions/524.html>
+- CWE 530 (Exposure of Backup File to an Unauthorized Control Sphere) - <https://cwe.mitre.org/data/definitions/530.html>
+- CWE 532 (Information Exposure Through Log Files) - <https://cwe.mitre.org/data/definitions/532.html>
+- CWE 534 (Information Exposure Through Debug Log Files) - <https://cwe.mitre.org/data/definitions/534.html>
+- CWE 634 (Weaknesses that Affect System Processes) - <https://cwe.mitre.org/data/definitions/634.html>
+- CWE 798 (Use of Hard-coded Credentials) - <https://cwe.mitre.org/data/definitions/798.html>
+- CWE 921 (Storage of Sensitive Data in a Mechanism without Access Control) - <https://cwe.mitre.org/data/definitions/921.html>
+- CWE 922 (Insecure Storage of Sensitive Information) - <https://cwe.mitre.org/data/definitions/922.html>
```
Comment (cpholguera) [inline Document-tr/0x07-V2-Data_Storage_and_Privacy_requirements.md:65]:
```suggestion
- CWE 922 (Insecure Storage of Sensitive Information) - <https://cwe.mitre.org/data/definitions/922.html>

```

File: Document-tr/0x08-V3-Cryptography_Verification_Requirements.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+- OWASP Mobile Top 10: M5 (Insufficient Cryptography) - <https://owasp.org/www-project-mobile-top-10/2016-risks/m5-insufficient-cryptography>
+- CWE 310 (Cryptographic Issues) - <https://cwe.mitre.org/data/definitions/310.html>
+- CWE 321 (Use of Hard-coded Cryptographic Key) - <https://cwe.mitre.org/data/definitions/321.html>
+- CWE 326 (Inadequate Encryption Strength) - <https://cwe.mitre.org/data/definitions/326.html>
+- CWE 327 (Use of a Broken or Risky Cryptographic Algorithm) - <https://cwe.mitre.org/data/definitions/327.html>
+- CWE 329 (Not Using a Random IV with CBC Mode) - <https://cwe.mitre.org/data/definitions/329.html>
+- CWE 330 (Use of Insufficiently Random Values) - <https://cwe.mitre.org/data/definitions/330.html>
+- CWE 337 (Predictable Seed in PRNG) - <https://cwe.mitre.org/data/definitions/337.html>
+- CWE 338 (Use of Cryptographically Weak Pseudo Random Number Generator (PRNG)) - <https://cwe.mitre.org/data/definitions/338.html>
```
Comment (cpholguera) [inline Document-tr/0x08-V3-Cryptography_Verification_Requirements.md:39]:
```suggestion
- CWE 338 (Use of Cryptographically Weak Pseudo Random Number Generator (PRNG)) - <https://cwe.mitre.org/data/definitions/338.html>

```

File: Document-tr/0x09-V4-Authentication_and_Session_Management_Requirements.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+Daha fazla bilgi için ayrıca bakınız:
+
+- OWASP Mobile Top 10: M4 (Insecure Authentication) - <https://owasp.org/www-project-mobile-top-10/2016-risks/m4-insecure-authentication>
+- OWASP Mobile Top 10: M6 (Insecure Authorization) - <https://owasp.org/www-project-mobile-top-10/2016-risks/m6-insecure-authorization>
+- CWE 287 (Improper Authentication) - <https://cwe.mitre.org/data/definitions/287.html>
+- CWE 307 (Improper Restriction of Excessive Authentication Attempts) - <https://cwe.mitre.org/data/definitions/307.html>
+- CWE 308 (Use of Single-factor Authentication) - <https://cwe.mitre.org/data/definitions/308.html>
+- CWE 521 (Weak Password Requirements) - <https://cwe.mitre.org/data/definitions/521.html>
+- CWE 604 (Use of Client-Side Authentication) - <https://cwe.mitre.org/data/definitions/604.html>
+- CWE 613 (Insufficient Session Expiration) - <https://cwe.mitre.org/data/definitions/613.html>
```
Comment (cpholguera) [inline Document-tr/0x09-V4-Authentication_and_Session_Management_Requirements.md:41]:
```suggestion
- CWE 613 (Insufficient Session Expiration) - <https://cwe.mitre.org/data/definitions/613.html>

```

File: Document-tr/0x10-V5-Network_communication_requirements.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+- CWE 296 (Improper Following of a Certificate's Chain of Trust) - <https://cwe.mitre.org/data/definitions/296.html>
+- CWE 297 (Improper Validation of Certificate with Host Mismatch) - <https://cwe.mitre.org/data/definitions/297.html>
+- CWE 298 (Improper Validation of Certificate Expiration) - <https://cwe.mitre.org/data/definitions/298.html>
+- CWE 308 (Use of Single-factor Authentication) - <https://cwe.mitre.org/data/definitions/308.html>
+- CWE 319 (Cleartext Transmission of Sensitive Information) - <https://cwe.mitre.org/data/definitions/319.html>
+- CWE 326 (Inadequate Encryption Strength) - <https://cwe.mitre.org/data/definitions/326.html>
+- CWE 327 (Use of a Broken or Risky Cryptographic Algorithm) - <https://cwe.mitre.org/data/definitions/327.html>
+- CWE 780 (Use of RSA Algorithm without OAEP) - <https://cwe.mitre.org/data/definitions/780.html>
+- CWE 940 (Improper Verification of Source of a Communication Channel) - <https://cwe.mitre.org/data/definitions/940.html>
+- CWE 941 (Incorrectly Specified Destination in a Communication Channel) - <https://cwe.mitre.org/data/definitions/941.html>
```
Comment (cpholguera) [inline Document-tr/0x10-V5-Network_communication_requirements.md:39]:
```suggestion
- CWE 941 (Incorrectly Specified Destination in a Communication Channel) - <https://cwe.mitre.org/data/definitions/941.html>

```

File: Document-tr/0x11-V6-Interaction_with_the_environment.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+- CWE 200 (Information Leak / Disclosure) - <https://cwe.mitre.org/data/definitions/200.html>
+- CWE 250 (Execution with Unnecessary Privileges) - <https://cwe.mitre.org/data/definitions/250.html>
+- CWE 672 (Operation on a Resource after Expiration or Release) - <https://cwe.mitre.org/data/definitions/672.html>
+- CWE 749 (Exposed Dangerous Method or Function) - <https://cwe.mitre.org/data/definitions/749.html>
+- CWE 772 (Missing Release of Resource after Effective Lifetime) - <https://cwe.mitre.org/data/definitions/772.html>
+- CWE 920 (Improper Restriction of Power Consumption) - <https://cwe.mitre.org/data/definitions/920.html>
+- CWE 925 (Improper Verification of Intent by Broadcast Receiver) - <https://cwe.mitre.org/data/definitions/925.html>
+- CWE 926 (Improper Export of Android Application Components) - <https://cwe.mitre.org/data/definitions/926.html>
+- CWE 927 (Use of Implicit Intent for Sensitive Communication) - <https://cwe.mitre.org/data/definitions/927.html>
+- CWE 939 (Improper Authorization in Handler for Custom URL Scheme) - <https://cwe.mitre.org/data/definitions/939.html>
```
Comment (cpholguera) [inline Document-tr/0x11-V6-Interaction_with_the_environment.md:45]:
```suggestion
- CWE 939 (Improper Authorization in Handler for Custom URL Scheme) - <https://cwe.mitre.org/data/definitions/939.html>

```

File: Document-tr/0x12-V7-Code_quality_and_build_setting_requirements.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+- CWE 95 (Improper Neutralization of Directives in Dynamically Evaluated Code ('Eval Injection')) - <https://cwe.mitre.org/data/definitions/95.html>
+- CWE 119 (Improper Restriction of Operations within the Bounds of a Memory Buffer) - <https://cwe.mitre.org/data/definitions/119.html>
+- CWE 215 (Information Exposure through Debug Information) - <https://cwe.mitre.org/data/definitions/215.html>
+- CWE 388 (7PK - Errors) - <https://cwe.mitre.org/data/definitions/388.html>
+- CWE 489 (Leftover Debug Code) - <https://cwe.mitre.org/data/definitions/489.html>
+- CWE 502 (Deserialization of Untrusted Data) - <https://cwe.mitre.org/data/definitions/502.html>
+- CWE 511 (Logic/Time Bomb) - <https://cwe.mitre.org/data/definitions/511.html>
+- CWE 656 (Reliance on Security through Obscurity) - <https://cwe.mitre.org/data/definitions/656.html>
+- CWE 676 (Use of Potentially Dangerous Function)  - <https://cwe.mitre.org/data/definitions/676.html>
+- CWE 937 (OWASP Top Ten 2013 Category A9 - Using Components with Known Vulnerabilities) - <https://cwe.mitre.org/data/definitions/937.html>
```
Comment (cpholguera) [inline Document-tr/0x12-V7-Code_quality_and_build_setting_requirements.md:42]:
```suggestion
- CWE 937 (OWASP Top Ten 2013 Category A9 - Using Components with Known Vulnerabilities) - <https://cwe.mitre.org/data/definitions/937.html>

```

File: Document-tr/0x15-V8-Resiliency_Against_Reverse_Engineering_Requirements.md [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+
+- Android: Testing Resiliency Against Reverse Engineering - <https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05j-Testing-Resiliency-Against-Reverse-Engineering.md>
+- iOS: Testing Resiliency Against Reverse Engineering - <https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md>
+
+Daha fazla bilgi için ayrıca bakınız:
+
+- OWASP Mobile Top 10: M8 (Code Tampering) - <https://owasp.org/www-project-mobile-top-10/2016-risks/m8-code-tampering>
+- OWASP Mobile Top 10: M9 (Reverse Engineering) - <https://owasp.org/www-project-mobile-top-10/2016-risks/m9-reverse-engineering>
+- OWASP Reverse Engineering Threats - <https://wiki.owasp.org/index.php/Technical_Risks_of_Reverse_Engineering_and_Unauthorized_Code_Modification>
+- OWASP Reverse Engineering and Code Modification Prevention - <https://wiki.owasp.org/index.php/OWASP_Reverse_Engineering_and_Code_Modification_Prevention_Project>
```
Comment (cpholguera) [inline Document-tr/0x15-V8-Resiliency_Against_Reverse_Engineering_Requirements.md:72]:
```suggestion
- OWASP Reverse Engineering and Code Modification Prevention - <https://wiki.owasp.org/index.php/OWASP_Reverse_Engineering_and_Code_Modification_Prevention_Project>

```

File: Document-tr/0x90-Appendix-A_Glossary.md [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+- **Threat Modeling** – Tehdit ajanlarını, güvenlik bölgelerini, güvenlik kontrollerini ve önemli teknik ve ticari varlıkları belirlemek için giderek artan şekilde iyileştirilmiş güvenlik mimarileri geliştirmekten oluşan bir teknik.
+- **Transport Layer Security** – İnternet üzerinden iletişim güvenliği sağlayan kriptografik protokoller.
+- **URI ve URL** – Bir URI, bir adı veya web kaynağını tanımlamak için kullanılan bir karakter dizisidir. URL genellikle bir kaynağa referans olarak kullanılır.
+- **User Acceptance Testing (UAT)** – Geleneksel olarak, tüm yazılım testlerinin yayınlanmadan önce gerçekleştirildiği üretim ortamı gibi davranan bir test ortamı.
+- **Verifier** – OWASP MASVS gereksinimlerine göre bir başvuruyu inceleyen kişi veya ekip.
+- **Whitelist** – İzin verilen veri veya işlemlerin listesi, örneğin giriş doğrulaması gerçekleştirmesine izin verilen karakterlerin listesi.
+- **X.509 Certificate** – X.509 sertifikası, genel anahtarın sertifikada bulunan kullanıcıya, bilgisayara veya hizmet kimliğine ait olduğunu doğrulamak için yaygın olarak kabul edilen uluslararası X.509 genel anahtar altyapısı (PKI) standardını kullanan dijital bir sertifikadır.
```
Comment (cpholguera) [inline Document-tr/0x90-Appendix-A_Glossary.md:42]:
```suggestion
- **X.509 Certificate** – X.509 sertifikası, genel anahtarın sertifikada bulunan kullanıcıya, bilgisayara veya hizmet kimliğine ait olduğunu doğrulamak için yaygın olarak kabul edilen uluslararası X.509 genel anahtar altyapısı (PKI) standardını kullanan dijital bir sertifikadır.

```

File: Document-tr/0x91-Appendix-B_References.md [hunk 14]
```diff
@@ -0,0 +1,14 @@
...
+- OWASP Mobile Security Testing Guide - [https://owasp.org/www-project-mobile-security-testing-guide/](https://owasp.org/www-project-mobile-security-testing-guide/)
+- OWASP Mobile Top 10 Risks - [https://owasp.org/www-project-mobile-top-10/](https://owasp.org/www-project-mobile-top-10/)
+- OWASP Reverse Engineering and Code Modification Prevention - [https://wiki.owasp.org/index.php/OWASP_Reverse_Engineering_and_Code_Modification_Prevention_Project](https://wiki.owasp.org/index.php/OWASP_Reverse_Engineering_and_Code_Modification_Prevention_Project)
+
+Benzer şekilde, aşağıdaki web siteleri bu standardın kullanıcıları/benimseyenler için büyük olasılıkla yararlıdır:
+
+- MITRE Common Weakness Enumeration - [http://cwe.mitre.org/](http://cwe.mitre.org/)
+- PCI Security Standards Council - [https://www.pcisecuritystandards.org](https://www.pcisecuritystandards.org)
+- PCI Data Security Standard (DSS) v3.0 Requirements and Security Assessment Procedures - [https://www.pcisecuritystandards.org/documents/PCI\_DSS\_v3.pdf](https://www.pcisecuritystandards.org/documents/PCI_DSS_v3.pdf)
```
Comment (cpholguera) [inline Document-tr/0x91-Appendix-B_References.md:14]:
```suggestion
- PCI Data Security Standard (DSS) v3.0 Requirements and Security Assessment Procedures - [https://www.pcisecuritystandards.org/documents/PCI\_DSS\_v3.pdf](https://www.pcisecuritystandards.org/documents/PCI_DSS_v3.pdf)

```

File: Document-tr/CHANGELOG.md [hunk 15]
```diff
@@ -0,0 +1,94 @@
...
+- Gereksinim 2.2 "Hassas veriler, uygulama alanı veya sistem kimlik depolama alanları dışındaki alanlarda saklanmamalıdır." eklendi.
+- Gereksinim 2.1, "Sistem kimlik bilgileri depolama olanakları, PII, kullanıcı kimlik bilgileri veya kriptografik anahtarlar gibi hassas verileri depolamak için uygun şekilde kullanılır." olarak yeniden ifade edildi.
+
+## V1.0 12 - Ocak 2018
+
+Aşağıdaki değişiklikler 1.0 sürümünün bir parçasıdır:
+
+- 8.12 ile aynı olan 8.9 silindi.
+- 4.6 daha genel hale getirildi.
+- Küçük düzeltmeler (yazım hataları vb.)
```
Comment (cpholguera) [inline Document-tr/CHANGELOG.md:94]:
```suggestion
- Küçük düzeltmeler (yazım hataları vb.)

```

File: Document-tr/GLOSSARY.md [hunk 16]
```diff
@@ -0,0 +1,161 @@
...
+
+OWASP MASVS gereksinimlerine göre bir başvuruyu inceleyen kişi veya ekip.
+
+## Whitelist
+
+İzin verilen veri veya işlemlerin listesi, örneğin giriş doğrulaması gerçekleştirmesine izin verilen karakterlerin listesi.
+
+## X.509 Certificate
+
+X.509 sertifikası, genel anahtarın sertifikada bulunan kullanıcıya, bilgisayara veya hizmet kimliğine ait olduğunu doğrulamak için yaygın olarak kabul edilen uluslararası X.509 genel anahtar altyapısı (PKI) standardını kullanan dijital bir sertifikadır.
```
Comment (cpholguera) [inline Document-tr/GLOSSARY.md:161]:
```suggestion
X.509 sertifikası, genel anahtarın sertifikada bulunan kullanıcıya, bilgisayara veya hizmet kimliğine ait olduğunu doğrulamak için yaygın olarak kabul edilen uluslararası X.509 genel anahtar altyapısı (PKI) standardını kullanan dijital bir sertifikadır.

```

File: Document-tr/SUMMARY.md [hunk 17]
```diff
@@ -0,0 +1,23 @@
...
+- [V4: Kimlik Doğrulama ve Oturum Yönetimi Gereksinimleri](0x09-V4-Authentication_and_Session_Management_Requirements.md)
+- [V5: Ağ İletişim Gereksinimleri](0x10-V5-Network_communication_requirements.md)
+- [V6: Platform Etkileşim Gereksinimleri](0x11-V6-Interaction_with_the_environment.md)
+- [V7: Kod Kalitesi ve Derleme Ayar Gereksinimleri](0x12-V7-Code_quality_and_build_setting_requirements.md)
+- [V8: Dayanıklılık Gereksinimleri](0x15-V8-Resiliency_Against_Reverse_Engineering_Requirements.md)
+
+## Ekler
+
+- [Ek A: Sözlük](GLOSSARY.md)
+- [Ek B: Referanslar](0x91-Appendix-B_References.md)
```
Comment (cpholguera) [inline Document-tr/SUMMARY.md:23]:
```suggestion
- [Ek B: Referanslar](0x91-Appendix-B_References.md)

```

**Review by cpholguera — CHANGES_REQUESTED** [review]

**Review by cpholguera — APPROVED** [review]
Excellent work @haktanemik! And apologies for the delay. We were focusing on the MASVS refactoring as you might know. Thank you very much!

**Comment by Glassware123** [comment]
Hi @cpholguera,

Thanks for great news! Have a nice day.

Best regards.

**Pull request merged**